pkg/controller/container-runtime-config/container_runtime_config_controller.go

```diff
@@ -213,6 +217,7 @@ func New(
 
 	ctrl.clusterVersionLister = clusterVersionInformer.Lister()
 	ctrl.clusterVersionListerSynced = clusterVersionInformer.Informer().HasSynced
+	ctrl.queue.Add(forceSyncOnUpgrade)
```

Comment:
Have you observed this to be necessary? The new controller is expected to re-gen all the MCs so it should always be synced post-new controller deployment